### PR TITLE
Add description tab to IE

### PR DESCRIPTION
**Pull request by ClevelandF**
This update adds a description tab to the Income Explorer tool.

This description tab:
- Explains the results that are presented in the IE, including descriptions of the included work incentive measures
- Outlines the tax and transfer schemes that are included in the tool
- Outlines limitations surrounding modelling of the accommodation supplement (asset tests are not included in the tool)
- Outlines the tax and transfer schemes that are not included in the tool

### CODE REVIEW

**Comment by YvonneWanggg** [comment]
The descriptions look good to me

**Review by YvonneWanggg — APPROVED** [review]

**Review by sijinzhang-tsy — APPROVED** [review]
Looks good to me, well done

File: ui.R [hunk 1]
```diff
@@ -172,6 +180,86 @@ shinyUI(fluidPage(
...
+              tags$li(strong("Replacement Rate (RR)"), " – The RR tab shows what percentage of an individual’s income in the hand would be replaced by government welfare transfers if they chose to stop working.",
+                      span(title = "If someone earned $1,000 per week at their job and receives $600 in unemployment benefits when they stop working, their replacement rate would be 60%. A higher replacement rate means people can receive benefits that are closer to their work income, which may reduce their incentives to remain in employment. Replacement rates are also used to evaluate the adequacy of social safety nets and to compare welfare systems across different countries or regions.",
+                           class = "tooltip-span",
+                           "Example"),
+                      "$$RR=\\frac{\\text{income in the hand when not working}}{\\text{income in the hand when working}}$$"),
+              tags$li(strong("Participation Tax Rate (PTR)"), " – The PTR tab shows the total amount of additional tax paid and welfare transfers forgone by an individual as they move from unemployment into work, as a percentage of gross income.",
```
Comment (Luke-Symes-Tsy) [inline ui.R:197]: Suggest this sentence, for more consistency with the EMTR & RR sentences.
```suggestion
              tags$li(strong("Participation Tax Rate (PTR)"), " – The PTR tab shows what percentage of gross earnings is not received in the hand when an individual moves from unemployment into work, due to additional tax paid and reduced government welfare transfers.",
```

File: ui.R [hunk 2]
```diff
@@ -172,6 +180,86 @@ shinyUI(fluidPage(
...
+                      span(title = "If someone could earn $1000 by working but would lose $400 in benefits and pay $200 in taxes, their participation tax rate would be 60% (losing $600 compared to a gross gain of $1000). A higher participation tax rate means people keep less of their earnings when they start working, which might discourage them from entering the workforce. This measure helps policymakers understand how tax and transfer systems together affect people's incentives to work versus remain on benefits.",
+                           class = "tooltip-span",
+                           "Example"),
+                      "$$PTR=1-\\frac{\\text{income in the hand from working}-\\text{income in the hand when not working}}{\\text{gross earnings from working}}$$"),
+              tags$li(strong("Income Composition"), " – The Income Composition tab shows how  earnings, taxes, and various government welfare transfers combine to give an individual’s net income.")
```
Comment (Luke-Symes-Tsy) [inline ui.R:202]:
```suggestion
              tags$li(strong("Income Composition"), " – The Income Composition tab shows how earnings, taxes, and various government welfare transfers combine to give an individual’s net income, and how this changes as their gross earnings change.")

```

File: ui.R [hunk 3]
```diff
@@ -172,6 +180,86 @@ shinyUI(fluidPage(
...
+            tags$ul(
+              tags$li(strong("Accommodation Supplement (AS)"), " – This tool assumes full
+                      take up of AS for all individuals with qualifying incomes and accommodation costs.
+                      However, AS eligibility is subject to additional tests, including asset tests, that are not currently included in the Income Explorer tool. To model an
+                      individual that is ineligible for/chooses not to take up AS, set their accommodation costs to zero."),
+              tags$li(strong("Jobseeker Support (JSS)"), " – This tool applies the ", em("Single, 25 years or over"), " JSS rate to 
+              all qualifying single people without children. It does not account for the different rates of JSS that are 
+              provided to people aged between 18 and 24 years older.")
+            ),
+            h3("The Income Explorer does not currently model:"),
```
Comment (Luke-Symes-Tsy) [inline ui.R:234]: We don't model Kiwisaver contributions (personal or employer) either.

**Review by Luke-Symes-Tsy — APPROVED** [review]
Looks good Fergus, just a couple of suggestions for you to consider.

File: ui.R (hunk 3 above)
Comment (ClevelandF) [reply ui.R:234]: Added

**Pull request merged**